src/segment-loader.js

```diff
@@ -1089,12 +1089,6 @@ export default class SegmentLoader extends videojs.EventTarget {
       return;
     }
 
-    if (segmentInfo.mediaIndex === this.playlist_.segments.length - 1 &&
```

Comment:
This moved to checkBuffer_ so that we don't log about requesting segmentInfo when we wont be. It also seems like it should belong there.